_pages/openid-connect.md

```diff
@@ -52,18 +69,34 @@ Users need to authorize OpenID Connect 1.0 clients individually. To present the
 
 ### Authorization Request
 
-```
+View example as <button data-example="pkce">PKCE</button><button data-example="private_key_jwt">private_key_jwt</button>
```

Comment:
Do we need to define some acronyms? Or are they already defined elsewhere in this doc? "PKCE" and "JWT" etc.